docs/java-application-development/development-environment-setup.md

```diff
@@ -1,183 +1,161 @@
 # Development Environment Setup
 
-This document describes how to set up the development environment for Kura, which consists of the following components:
-
-* JVM (Java JDK SE 8)
-
-* Eclipse IDE
-
-* Kura Workspace setup
+In this document we'll cover the required steps to setup the Development Environment for contributing to the Eclipse Kura project. If, instead you want to develop applications or bundles running on Kura refer to the [Kura Workspace setup guide](./kura-workspace-setup.md).
 
 The Kura development environment may be installed on Windows, Linux, or Mac OS. The setup instructions will be the same across each OS though each system may have unique characteristics.
+
 !!! info
     The local emulation of Kura code is only supported in Linux and Mac, not in Windows.
 
-## JVM Installation
-
-Download and install JDK SE 8 from the following links as appropriate for your OS.
-
-For Windows and Linux users, the JDK can be downloaded from the following link: [Java SE 8 Downloads](https://adoptium.net/temurin/releases/?version=8).  Use the latest version of Java SE Development Kit and download the version appropriate for your system.
+This document will cover the use of Eclipse Oomph installer which is the easiest way to install and configure the Eclipse IDE to start contributing to Kura.
 
-For additional information regarding the installation of Java 8 on all supported operating systems, see [JDK 8 and JRE 8 Installation Guide](https://adoptium.net/installation).
+The setup requires three basic steps:
 
+1. Requirements installation
+2. Eclipse Oomph setup
+3. Kura maven build
```

Comment:
> Kura maven build

Eclipse Kura maven build

Reply:
Addressed in 07b2d2eb82c3658a32602d3b85954f5ac5c6d23e